### PR TITLE
T21 - Swapping of two variables.

### PR DESCRIPTION
**Pull request by Priyanka15-12**

### CODE REVIEW

File: Program.cs [hunk 1]
```diff
@@ -1,7 +1,15 @@
...
+   internal class Program {
+      static void Main (string[] args) {
+         Console.Write ("Enter a number for a= ");
+         int a = int.Parse (Console.ReadLine ());
+         Console.Write ("Enter a number for b= ");
+         int b = int.Parse (Console.ReadLine ());
+         swap (ref a, ref b);
+         Console.Write($"After swapping,\na = {a}\nb = {b}");
+      }
+      static void swap (ref int x, ref int y) {
```
Comment (MetaNaveen) [inline Program.cs:11]: You could use Single-line lambda expression.

**Review by MetaNaveen — CHANGES_REQUESTED** [review]

File: Program.cs (hunk 1 above)
Comment (MetaNaveen) [inline Program.cs:11]: Coding style not maintained.

**Review by MetaNaveen — CHANGES_REQUESTED** [review]

File: Program.cs (hunk 1 above)
Comment (Priyanka15-12) [reply Program.cs]: Modified with Lambda expression

File: Program.cs [hunk 2]
```diff
@@ -1,7 +1,13 @@
...
-            Console.WriteLine ("Hello, World!");
-        }
-    }
+   internal class Program {
+      static void Main (string[] args) {
+         Console.Write ("Enter a number for a = ");
+         int a = int.Parse (Console.ReadLine ());
+         Console.Write ("Enter a number for b = ");
+         int b = int.Parse (Console.ReadLine ());
+         void Swap (int x, int y) => (a, b) = (b, a);
```
Comment (MetaNaveen) [inline Program.cs:8]: Incorrect usage of method with unused parameter of x and y.
Method result is incorrect as if I use Swap(100, 100), a and b still gets swapped.
Read [ref](https://www.w3schools.blog/c-sharp-call-by-reference).

**Review by MetaNaveen — CHANGES_REQUESTED** [review]

**Comment by MetaNaveen** [comment]
Working fine.

**Final notes:**
You could have used "Top Level Statements" for the brevity of this solution.

**Pull request closed without merging**